Integrations/integration-ArcSightXML.yml

```diff
@@ -0,0 +1,193 @@
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Directory from which to get XML files and create incidents.
+  name: inputDirPath
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Directory to which put command XML files.
+  name: commandsDirPath
+  defaultvalue: ""
+  type: 0
+  required: true
+script:
+  script: |+
+    import glob
+    import xml.etree.ElementTree as ET
+    import lxml.etree
+    import datetime
+    import json
+    import os
+
+    SECURITY_EVENT_START_TAG = '<SecurityEvent id="%s"'
+    SECURITY_EVENT_END_TAG = '</SecurityEvent>'
+
+    LIST_CASE_ENTRIES = ['caseEvents', 'childOf', 'ownedBy']
+    TIME_CASE_ENTRIES = ['detectionTime', 'estimatedStartTime']
+
+    SYSTEM = ''
+    URI_PREFIX = ''
+    TOREPLACE = '\\"\\<\\>\\(\\)'
+
+    XML_TEMPLATE = """<?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE archive SYSTEM "../../schema/xml/archive/arcsight-archive.dtd">
+    <archive buildVersion="6.9.1.2195.0" buildTime="2-9-2016_19:0:8" createTime="{0}">
+        <ArchiveCreationParameters>
+            <action>insert</action>
+            <format>xml.external.case</format>
+            <include>
+                <list>
+                    <ref type="Case" uri="{2}" id="{1}"/>
+                </list>
+            </include>
+        </ArchiveCreationParameters>
+        <Case id="{1}" name="{3}" action="insert" >
+            <stage>{4}</stage>
+        </Case>
+    </archive>"""
+
+    FILENAME_TEMPLATE = 'ExternalEventTrackingData_{0}.xml'
+
+
+    def ref_to_dict(item):
+        return {'type': item.get('type'), 'uri': item.get('uri'), 'id': item.get('id')}
+
+
+    def build_case_data(case):
+        case_data = {case_child.tag: case_child.text for case_child in case}
+
+        # fix timestamp entries
+        for name in TIME_CASE_ENTRIES:
+            if name in case_data:
+                case_data[name] = str(datetime.datetime.fromtimestamp(int(case_data[name]) / 1000))
+
+        # fix list entries
+        for name in LIST_CASE_ENTRIES:
+            list_entry = case.find(name)
+            if list_entry is not None:
+                case_data[name] = json.dumps([ref_to_dict(item) for item in list_entry[0]])
+
+        # convert to a list of 'type' and 'value'
+        case_data = [{'type': key, 'value': value} for key, value in case_data.iteritems()]
+
+        return case_data
+
+
+    def create_incidents(all_labels, all_details):
+        incidents = []
+        for labels, details in zip(all_labels, all_details):
+            incident = {
+                'Name': 'ArcSight File Event',
+                'Labels': labels,
```

Comment:
There is no specification so I just put all, as we discussed. Please see the attached screenshots. I can't think of any field that I know is certainly redundant. I think that it's better to keep all of them. What do you think?
If you can specify exactly which fields you think should be removed it will be best.
![image](https://user-images.githubusercontent.com/26041559/26925972-65b65230-4c54-11e7-8e7a-2ce4dce0c901.png)
![image](https://user-images.githubusercontent.com/26041559/26925975-68fa8f92-4c54-11e7-9380-e331301b3910.png)